libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Memory/Expressions/DialogExpression.cs

```diff
@@ -67,5 +67,15 @@ public override void SetValue(object value)
 
             base.SetValue(value);
         }
+
+        public new string ToString()
```

Comment:
Why is the base base class, ExpressionProperty<T> using `public new string ToString()` instead of override? Wouldn't it be healthier to use override in ExpressionProperty<T> and here?